pkg/gcv/insight.go

```diff
@@ -0,0 +1,39 @@
+package gcv
+
+import "time"
+
+// Insight is modeled after the cloud recommender insight
+type Insight struct {
+	// Name omitted, will be added post-facto
+	Name string `json:"name,omitempty"`
+
+	// Description is a human readable summary
+	Description string `json:"description,omitempty"`
+
+	// TargetResources
+	TargetResources []string `json:"target_resources,omitempty"`
+
+	// InsightSubtype
+	InsightSubtype string `json:"insight_subtype,omitempty"`
+
+	// Content
+	Content map[string]interface{} `json:"content,omitempty"`
+
+	// LastRefreshTime
+	LastRefreshTime time.Time `json:",omitempty"` //omitted, will be added as part of job param
+
+	// ObservationPeriod
+	ObservationPeriod time.Duration `json:",omitempty"` // omitted, will be added as part of job param
+
+	// StateInfo
+	StateInfo StateInfo `json:",omitempty"` // omitted, will be added post-facto in DB field.
+
+	// Category
+	Category string `json:",omitempty"` //omitted, will always be SECURITY for forseti scans.
+}
+
+// StateInfo is the state of the data, not populated here.
```

Comment:
I'm kinda confused by the "no populated here" comment; add an explanation?

Reply:
Should be clearer now